rules/windows-powershell-commands.data

```diff
@@ -1,6 +1,6 @@
 # Sources:
 # Microsoft PowerShell Docs: https://github.com/MicrosoftDocs/PowerShell-Docs
-# - curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/MicrosoftDocs/PowerShell-Docs/git/trees/main\?recursive\=1 | jq -r '.tree[] .path | capture("reference/\\d.\\d/(.*)/(?<fn>[A-Z]\\w+-\\w+).md") | .fn' | sort | uniq
+# - curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/MicrosoftDocs/PowerShell-Docs/git/trees/main | jq -r '.tree[] .path | capture("reference/\\d.\\d/(.*)/(?<fn>[A-Z]\\w+-\\w+).md") | .fn' | sort | uniq
```

Comment:
```suggestion
# - curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/MicrosoftDocs/PowerShell-Docs/git/trees/main\?recursive\=1 | jq -r '.tree[] .path | capture("reference/\\d.\\d/(.*)/(?<fn>[A-Z]\\w+-\\w+).md") | .fn' | sort | uniq
```
Without recursive won't work.

Reply:
Fixed in [f779e7a](https://github.com/coreruleset/coreruleset/pull/3232/commits/f779e7a7a72362783de23524a5af95303db81d2f).